Osmalyzer/Data/Misc Data Fetchers/OsmNamesAnalysisData.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+
+namespace Osmalyzer;
+
+[UsedImplicitly]
+public class OsmNamesAnalysisData : AnalysisData
```

Comment:
I renamed this to StreetNameQualifiersAnalysisData to be "street name qualifier(s)" - since this only contains street names and not other names and only their qualifiers/descriptors, as well as not necessarily OSM-specific. I don't know if you intended to add different things here, although I prefer if we made new data files/types in that case. And we can rename back if it becomes more broad.

Reply:
I named this with a thought of extending this checks to other objects, not only streets, in mind. But it is completely reasonable to have it named your way. Let's if and when we'll implement any new item check how to extend that class.

Reply:
While this is more of an coincidence, the Latvian qualifiers are taken from `FuzzyAddressMatcher._suffixes`. I am not entirely sure how to "connect" the two to avoid duplicating data.

Reply:
Ahh, yeah, that's interesting problem. Then we can have all address suffixes in this file, and if we'll need more nomenclature items for later, we'll add them into a separate tsv. And will join everything in the fetcher class.